cypress/TestCases/FireboltCertification/LifecycleTestcases/LifecycleBackground.feature

```diff
@@ -14,7 +14,12 @@ Feature: Lifecycle_Background
             | state      |
             | foreground |
 
-        @notSupported
+    @notSupported
```

Comment:
why have we moved them to separate outlines ?

Reply:
If I don't move it to separate outline, the inactive example always gets the requiresPlatformImplementation tag in the report even though it has a notSupported tag.
<img width="592" alt="multiple examples" src="https://github.com/user-attachments/assets/58299726-310f-4ad5-ba83-0a37d6c1bda8">